hardware/areca.py

```diff
@@ -147,6 +150,13 @@ def _disable_password():
 
 def detect():
     """Detect Areca controller configuration."""
+    if not detect_utils.which("cli64"):
+        print("Cannot find cli64 binary")
```

Comment:
has to be `sys.stderr.write` (see below) or `print(..., file=sys.stderr)`. otherwise your message will mix into the output.

Reply:
yeah, I was a bit distracted when I did this, thanks!